packages/gamepads_ios/ios/gamepads_ios.podspec

```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint gamepads_ios.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'gamepads_ios'
+  s.version          = '0.0.1'
```

Comment:
should this be 0.1.1?

Reply:
This is actually 0.0.1 on gamepads_darwin too